protos/bottle/account/v1/events.proto

```diff
@@ -23,6 +23,12 @@ message TwoFactorRequested {
   string token = 2;
 }
 
+message TwoFactorRecoveryCodeUsed {
+  bottle.account.v1.User user = 1;
+  string recovery_code = 2;
+  int32 codes_remaining = 3;
```

Comment:
@system76/web-engineering I'm open to change this up my initial thought was we tell them which code was used and how many they have remaining.

Reply:
I'm good with that